src/SamplesApp/UITests.Shared/Windows_UI_ViewManagement/IsScreenCaptureEnabledTests.xaml

```diff
@@ -0,0 +1,16 @@
+﻿<UserControl
+    x:Class="UITests.Shared.Windows_UI_ViewManagement.IsScreenCaptureEnabledTests"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UITests.Shared.Windows_UI_ViewManagement"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    d:DesignHeight="300"
+    d:DesignWidth="400">
+
+    <StackPanel>
+		<CheckBox Content="Is screen capture enabled" x:Name="CaptureCheckBox" IsThreeState="False" Checked="CaptureCheckBoxChanged" Unchecked="CaptureCheckBoxChanged" />
```

Comment:
Same content for "Checked" and "Unchecked" ?

Reply:
Yes, wanted to have a single method as the handler - as the code would be exactly the same, just with true/false. In the end it is probably the same number of lines though :-D .

Reply:
Ok I see your point ;)